src/utils/responses.ts

```diff
@@ -26,6 +27,14 @@ export function ResponseNoContent(): IResponseNoContent {
   };
 }
 
+/**
+ * A response error for when a feature has been dismissed.
+ */
+export const ResponseErrorDismissed = ResponseErrorNotFound(
```

Comment:
this response type should be shared with the App. Maybe must be included in `io-ts-commons`?

Reply:
We definitely want to map more specific response like this in `io-ts-commons`. However, there's no point for this scenario as we aren't mapping anything, we are just passing a descriptive string.